build.gradle

```diff
@@ -1 +1,34 @@
-ant.importBuild 'build.xml'
\ No newline at end of file
+apply plugin: 'java'
+apply plugin:'application'
+mainClassName = "edu.ucsb.cs56.projects.utilities.credit_card_validator"
+
```

Comment:
On the following line add this:

`sourceSets.main.java.srcDirs` = ['src']

This says "Hey gradle, go look for my main class specified above in the `src` directory"

By default, gradle looks for a directory named `src/main/java/` for the start of your class path.